script.js

```diff
@@ -7,15 +8,180 @@ const menuLinks = document.querySelector('.navmenu');
 });
-
+// click to appear
 closeIcon.addEventListener('click', () => {
   menuLinks.classList.remove('change');
 });
 
 const listItems = document.querySelectorAll('.nav_item');
-
+// click to disappear
 listItems.forEach((element) => {
   element.addEventListener('click', () => {
     menuLinks.classList.remove('change');
   });
 });
+
+// project cards object
+const projects = [
+  {
+    projectId: 0,
+    title: 'Printing Data One',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    featuredImage: 'images/Background.png',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://have-samuel.github.io/portfolio1/',
+    sourceLink: 'https://github.com/Have-Samuel/portfolio1/tree/popup-menu',
+  },
+  {
+    projectId: 1,
+    title: 'Printing Data Two',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    featuredImage: 'images/Background.png',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://have-samuel.github.io/portfolio1/',
+    sourceLink: 'https://github.com/Have-Samuel/portfolio1/tree/popup-menu',
+  },
+  {
+    projectId: 2,
+    title: 'Printing Data Three',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    featuredImage: 'images/Background.png',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://have-samuel.github.io/mobile-ver/',
+    sourceLink: 'https://github.com/Have-Samuel/portfolio1/tree/popup-menu',
+  },
+];
+// Calling the cards
+const sectionCards = document.querySelector('.section-cards');
+
+function createProjectCard(project) {
+  const section = document.createElement('section');
+  section.classList = 'section1';
+  // <h4>
+  const projectHeading = document.createElement('h4');
+  projectHeading.innerHTML = project.title;
+  section.appendChild(projectHeading);
+  // <p>
+  const projectDesciption = document.createElement('p');
+  projectDesciption.textContent = project.description;
+  section.appendChild(projectDesciption);
+  // skills
+  const projectSkills = document.createElement('ul');
+  const skills = project.technologies;
+  // looping the skills
+  for (let i = 0; i < skills.length; i += 1) {
+    const projectSkillsListItem = document.createElement('li');
+    projectSkillsListItem.textContent = skills[i];
+    projectSkills.appendChild(projectSkillsListItem);
+    section.appendChild(projectSkills);
+  }
+  // <button>
+  const projectButton = document.createElement('button');
+  projectButton.classList = 'see-button';
+  projectButton.innerText = 'See project';
+
+  section.appendChild(projectButton);
+  // when clicked shows Id
+  projectButton.addEventListener('click', () => {
+    console.log(project.projectId);
```

Comment:
According to general Javascript best practices requirements:

![image](https://user-images.githubusercontent.com/57421823/156448371-13fdc710-fe81-4c2b-962d-93e0e2aa58ea.png)

Therefore:

- [ ] Please, kindly do not push to the repo any `console.log` calls.